eng/Baseline.xml

```diff
@@ -1,91 +1,89 @@
 ﻿<!--
 
-This file contains a list of all the packages and their versions which were released in ASP.NET Core 3.0.0. 
+This file contains a list of all the packages and their versions which were released in ASP.NET Core 3.1.0.
 Update this list when preparing for a new patch.
 
 -->
-
-<Baseline Version="3.0.0">
+<Baseline Version="3.1.0">
```

Comment:
@aspnet/build could you sign off on this?

Reply:
Makes sense to me.